README.md

```diff
@@ -41,7 +41,7 @@ import createContainer from "constate";
 // 1️⃣ Create a custom hook as usual
 function useCounter() {
   const [count, setCount] = useState(0);
-  const increment = () => setCount(count + 1);
+  const increment = () => setCount(count => count + 1);
```

Comment:
Thank you @jamesplease! Could you please change the variable name so it doesn't conflict?
```suggestion
  const increment = () => setCount(prevCount => prevCount + 1);
```

Reply:
Oh, I didn't see I could do that too haha